cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -114,7 +115,9 @@ <h3 >
             {{ parse_links(page.answer_base.next_step.text) | safe }}
         </div>
     {% endif %}
-
+    {% if page.sidebar_panels %}
+            {{ streamfield_sidefoot.render(page.sidebar, half_width=true) }}
```

Comment:
You can remove the `half_width=true` property.

Reply:
Thanks!